### DISCUSSION TITLE
How to set up proxy for actions-runner-controller to access the GitHub API

### DISCUSSION THREAD

**Discussion by pixiake** [actions/actions-runner-controller#3081] [Questions]
![image](https://github.com/actions/actions-runner-controller/assets/22290449/91f55780-794e-4618-a7c8-35d59d15f02f)
![image](https://github.com/actions/actions-runner-controller/assets/22290449/ecd53be1-2a93-4c7c-9c74-a1900faa8a14)

When I run ARC, I found that accessing api.github.com in the pod would time out. However, even after setting a proxy env for the pod, it still reports connection timeout. How can I configure it so that ARC can make requests to api.github.com through the proxy when running? Thanks

**Comment by vinnamkim** [comment]
Hi @pixiake  did you solve your problem?

**Comment by manvindar-telus** [comment]
any update on the fix?
